src/style.css

```diff
@@ -0,0 +1,285 @@
+  color: #121212;
+}
+
+.book--info h1 {
+  width: 12.313rem;
+  height: 1.813rem;
+  margin: 0.188rem 20.188rem 0 0;
+  font-size: 1.375rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.2px;
+  color: #121212;
+}
+
+.book--info p {
+  width: 6.688rem;
+  height: 1.188rem;
+  margin: 0.25rem 0 0;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+}
+
+.book--utils {
+  list-style: none;
+  display: flex;
+  gap: 10px;
+}
+
+.left--right {
+  border-left: 1px solid #e8e8e8;
+  border-right: 1px solid #e8e8e8;
+}
+
+button {
+  background: none;
+  border: none;
+  padding: 6px;
+  border-radius: 2px;
+}
+
+.book--utils--button {
+  padding: 0.5rem;
+  cursor: pointer;
+  width: 4.563rem;
+  height: 2.188rem;
+  margin: 0.188rem 0.938rem 0.125rem 0;
+  font-size: 0.875rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #4386bf;
+}
+
+.book--utils button:hover {
+  background-color: #d7e1ec;
+}
+
+.loader {
+  width: 4.25rem;
+  height: 3.25rem;
+}
+
+.chapter h4 {
+  width: 7.875rem;
+  height: 1.125rem;
+  margin: 0.063rem 5.375rem 0 3.688rem;
+  opacity: 0.5;
+  font-size: 0.813rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #121212;
+}
+
+.chapter h5 {
+  width: 13.25rem;
+  height: 1.313rem;
+  margin: 0.438rem 0 0.25rem 3.688rem;
+  font-size: 1rem;
+  font-weight: 300;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.4px;
+  color: #121212;
+}
+
+.chapter--button {
+  width: 11.5rem;
+  height: 2.063rem;
+  margin: 0.75rem 1.75rem 0.563rem 3.688rem;
+  padding: 0.438rem 1.188rem 0.5rem 1.375rem;
+  border-radius: 3px;
+  background-color: #0290ff;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  margin: 2.313rem 6.25rem 0;
+  gap: 2rem;
+  justify-content: center;
+  padding: 2rem;
+  border-top: 4px solid #e8e8e8;
+}
+
+.form span {
+  width: 10.875rem;
+  height: 1.5rem;
+  margin: 1.813rem 32.75rem 1.188rem 0;
+  font-size: 1.25rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+.inner--form {
+  display: grid;
+  grid-template-columns: 2fr 1fr 1fr 1fr;
+  flex-direction: row;
+  gap: 1.5rem;
+}
+
+.author {
+  width: 450px;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+}
+
+.submit {
+  width: 200px;
+  border-radius: 3px;
+  background-color: red;
+  border: none;
+}
+
+input {
+  padding: 0.5rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+}
+
+.dropdown {
+  width: 150px;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+}
```

Comment:
- [ ] Good job so far with your project design :+1:  But your app design is a bit far from the Bookstore design in [Zeplin](https://zeplin.io/). :cry:  Please make sure that your app design is Styling EXACTLY like Zeplin Bookstore's design, use the same styling element provided. As a developer, it's very important to peer attention to details cause your future employer will judge your work on it. And following the zeplin design will make your application works fine and wonderful :smile: 
Please check the screenshots below to see the differences :arrow_down: 

#### Your design
![Screenshot from 2022-08-27 12-28-59](https://user-images.githubusercontent.com/6615394/187030636-1e80a262-d59f-4fee-aed7-a790c5b2b8b0.png)
![Screenshot from 2022-08-27 12-29-15](https://user-images.githubusercontent.com/6615394/187030634-e9d94492-244a-47d7-bef4-a50eaf75260e.png)


#### Zeplin design
![Screenshot from 2022-08-27 12-28-29](https://user-images.githubusercontent.com/6615394/187030638-2505a940-ea42-41eb-aed4-c4c1939fbfd0.png)
![Screenshot from 2022-08-27 12-28-41](https://user-images.githubusercontent.com/6615394/187030637-667720ed-3fc8-4a73-9fb6-0d646f392918.png)